### ISSUE TITLE
Invalid Link

### ISSUE THREAD

**Issue by sharmapradyumn** [coala/coala-bears#2955]
### There is a issue in coala'a web page,In About section  the "Bears Documentation" button is not giving expected result of Bears Documentation.

# Steps to Reproduce

1.check the link of "Bears Documentation" button

# Actual Results
 when we click on "Bears Documentation" its redirect on  "LANGUAGES" section of website 

# Expected Results
 when we click on "Bears Documentation" its must redirected on Documentation page of Bears.

**Comment by abhishalya** [comment]
This is not the correct place for this issue, but anyways it has been already reported. Have a look at https://github.com/coala/coala-bears/issues/2632

**Comment by sharmapradyumn** [comment]
can you tell  me the correct place for this type of issue

**Comment by KVGarg** [comment]
@abhishalya I don’t think that we do a coala-bears api documentation as like coala repository has. Currently, we are having the coala-bears documentation on coala Webservices which we fetch and display it under Languages tab ( Broken for now )

**Comment by KVGarg** [comment]
> can you tell me the correct place for this type of issue

Correct Place: https://github.com/coala/landing-frontend
Also an issue related to this has already been created. We are using coala Webservices for displaying the bears metadata documentation NOT the code

**Comment by mitchell-frost** [comment]
Can I work on this?

**Comment by abhishalya** [comment]
@mitchell-frost No, this is related to the webservices. We might need someone experienced to fix that whole thing. 

@KVGarg can we fix the webservices? I'm not sure what is required. Iirc, it was a data issue..

**Comment by KVGarg** [comment]
Maybe there was some issues with docker container not having coala bears, or the issue was something else. I didn’t remember completely as it has been long time I have look on webservices. 
Maybe this PR was a solution to that, https://gitlab.com/coala/landing/-/merge_requests/63. 
I’m not getting enough time to get all the enhancements and improvements merged to coala so that newcomers can use it. Will try this week to get it done.

**Comment by abhishalya** [comment]
@KVGarg will check that our over weekend, if that PR is what we require then we should get it merged.
